src/Eshopworld.Messaging/TopicAdapter.cs

```diff
@@ -180,5 +183,29 @@ protected override void Dispose(bool disposing)
 
             base.Dispose(disposing);
         }
+
+        private static void CheckTopicName(string topicName)
+        {
+            if (topicName.Length > 260) // SB quota: Entity path max length
+            {
+                throw new InvalidOperationException(
+                    $@"You can't create queues for the topic name {topicName} because it exceeds 260 characters.");
+            }
+        }
+
+        private static string CheckTopicType()
+        {
+            var topicType = typeof(T);
+            var topicName = topicType.GetEntityName();
+
+            if (topicName.Length > 260) // SB quota: Entity path max length

```

Comment:
We are repeating this topic name length check in these 2 methods. We can change `CheckTopicType` to `GetTopicName` and remove the length check from it. Then we can have length check only in `CheckTopicName` method

Reply:
yes, but the error messages are different. That's why I kept both checks